### PR TITLE
Update dev

### PR DESCRIPTION
**Pull request by ksenia-krasheninnikova**
The changes from the following commits were merged:

[Template update for nf-core/tools version 2.5](https://github.com/sanger-tol/genomeassembly/pull/2/commits/372b36c76559e23dbf58e0eb70d341bfb036d5ef)
[Remove test.nf](https://github.com/sanger-tol/genomeassembly/pull/2/commits/21293c7f2ce8c44c861b79964cc0739df6fc4889)

Also, PR contains fix to the assets tests file and the workflow name update.

Changes were tested with
`nextflow run main.nf -profile test,singularity --outdir test -c conf/test.config`

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/genomeassembly/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by muffato — CHANGES_REQUESTED** [review]
I noticed several changes that replace `sanger-tol/` with `nf-core/sanger-tol-`. `sanger-tol/` was fine !

**Comment by priyanka-surana** [comment]
@ksenia-krasheninnikova Can you make the change to `nf-core/sanger-tol` to `sanger-tol`? I think this happened when template update was done.

File: main.nf [hunk 1]
```diff
@@ -23,13 +25,13 @@ WorkflowMain.initialise(workflow, params, log)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-include { GENOMEASSEMBLY } from './workflows/genomeassembly'
+include { SANGER_TOL_GENOMEASSEMBLY } from './workflows/sanger-tol-genomeassembly'
```
Comment (priyanka-surana) [inline main.nf:28]: I think same goes for this. Revert back to the old convention.

File: workflows/genomeassembly.nf [hunk 2]
```diff
@@ -6,9 +6,6 @@
 
 def summary_params = NfcoreSchema.paramsSummaryMap(workflow, params)
 
-// Validate input parameters
-WorkflowGenomeassembly.initialise(params, log)
```
Comment (muffato) [inline workflows/genomeassembly.nf:10]: Is that really needed ? I wonder if this line was failing because the file has been renamed

**Review by muffato — COMMENTED** [review]
`lib/WorkflowSanger-tol-genomeassembly.groovy` can be renamed back as well (and the name of the class inside as well)

**Review by priyanka-surana — APPROVED** [review]
template changes mostly

**Review by muffato — APPROVED** [review]
I've run `black` and updated a few things like the copyright statement, the branding, etc.

**Pull request merged**